AzureCalling/app/src/main/res/layout/participant_view.xml

```diff
@@ -29,22 +29,22 @@
             android:layout_marginBottom="3dp"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent">
+
             <TextView
                 android:id="@+id/display_name"
                 android:layout_width="wrap_content"
-                android:maxLines="1"
-                android:inputType="textNoSuggestions"
+                android:layout_height="wrap_content"
```

Comment:
android:inputType="textNoSuggestions" appears to be removed. Would this cause the underline issue for invalid words?

Reply:
Returned it back